src/styles/index.scss

```diff
@@ -2,2 +1,135 @@
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$second-color: #616070;
+
+body,
+p {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  width: 200px;
+  align-items: center;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    padding-top: 32px;
+    padding-left: 19px;
+    padding-right: 19px;
+    margin-bottom: 40px;
+  }
+  &__name {
+    width: 166px;
+    height: 36px;
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+    text-align: left;
+  }
+  &__code {
+    width: 98px;
+    height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding: 0 16px;
+    margin-bottom: 16px;
+    color: $second-color;
+    text-align: left;
```

Comment:
The padding here seems to be applied only to the left and right. If you intend to center the text inside a block, consider using `text-align: center;` instead of padding for horizontal centering, unless there's a specific design requirement.